src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -263,7 +262,7 @@ public Path getFulltextIndexPath() {
 
         if (getDatabasePath().isPresent()) {
             Path databasePath = getDatabasePath().get();
-            String fileName = BackupFileUtil.getUniqueFilePrefix(databasePath) + "--" + databasePath.getFileName();
+            String fileName = BackupFileUtil.getUniqueFilePrefix(databasePath) + "-" + databasePath.getFileName();
```

Comment:
I wanted to update the index path from JabRef v5 to v6 to ensure the index is correctly updated

Reply:
Maybe instead of changing the path name to distinguish between versions, we can create a version to the Lucene index in the metadata. When this version is updated, the index should be rebuilt by removing the old one, creating a new one, and updating the version. 
The version const should be updated whenever changes are made to the index, such as renaming fields or changing the analyzer.
If you agree, should this version be stored in the metadata or somewhere else?